packages/grafana-ui/src/themes/GlobalStyles/forms.ts

```diff
@@ -27,5 +27,345 @@ export function getFormElementStyles(theme: GrafanaTheme2) {
       {
         width: 'auto', // Override of generic input selector
       },
+    '.gf-form': {
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'flex-start',
+      textAlign: 'left',
+      position: 'relative',
+      marginBottom: theme.spacing(0.5),
+
+      '&--offset-1': {
+        marginLeft: theme.spacing(2),
+      },
+
+      '&--grow': {
+        flexGrow: 1,
+      },
+
+      '&--flex-end': {
+        justifyContent: 'flex-end',
+      },
+
+      '&--align-center': {
+        alignContent: 'center',
+      },
+
+      '&--alt': {
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+
+        '.gf-form-label': {
+          padding: '4px 0',
+        },
+      },
+    },
+    '.gf-form--has-input-icon': {
+      position: 'relative',
+      marginRight: theme.spacing(0.5),
+
+      '.gf-form-input-icon': {
+        position: 'absolute',
+        top: '8px',
+        fontSize: theme.typography.size.lg,
+        left: '10px',
+        color: theme.colors.text.disabled,
+      },
+
+      '> input': {
+        paddingLeft: '35px',
+
+        '&:focus + .gf-form-input-icon': {
+          color: theme.colors.text.secondary,
+        },
+      },
+
+      '.Select--multi .Select-multi-value-wrapper, .Select-placeholder': {
+        paddingLeft: '30px',
+      },
+    },
+
+    '.gf-form-disabled': {
+      color: theme.colors.text.secondary,
+
+      '.gf-form-select-wrapper::after': {
+        color: theme.colors.text.secondary,
+      },
+
+      'a, .gf-form-input': {
+        color: theme.colors.text.secondary,
+      },
+    },
+
+    '.gf-form-group': {
+      marginBottom: theme.spacing(5),
+    },
+    '.gf-form-inline': {
+      display: 'flex',
+      flexDirection: 'row',
+      flexWrap: 'wrap',
+      alignContent: 'flex-start',
+
+      '&--nowrap': {
+        flexWrap: 'nowrap',
+      },
+
+      '&--xs-view-flex-column': {
+        flexDirection: 'row',
+        flexWrap: 'nowrap',
+        [theme.breakpoints.down('sm')]: {
+          flexDirection: 'column',
+        },
+      },
+
+      '.select-container': {
+        marginRight: theme.spacing(0.5),
+      },
+
+      '.gf-form-spacing': {
+        marginRight: theme.spacing(0.5),
+      },
+    },
+
+    '.gf-form-button-row': {
+      paddingTop: theme.spacing(3),
+      'a, button': {
+        marginRight: theme.spacing(2),
+      },
+    },
+    '.gf-form-label': {
+      display: 'flex',
+      alignItems: 'center',
+      padding: theme.spacing(0, 1),
+      flexShrink: 0,
+      fontWeight: theme.typography.fontWeightMedium,
+      fontSize: theme.typography.size.sm,
+      backgroundColor: theme.colors.background.secondary,
+      height: '32px',
+      lineHeight: '32px',
+      marginRight: theme.spacing(0.5),
+      borderRadius: theme.shape.radius.default,
+      justifyContent: 'space-between',
+      border: 'none',
+
+      '&--grow': {
+        flexGrow: 1,
+      },
+
```

Comment:
Don't we have to transform the following one?
```jsx
 &--error {
    color: $critical;
  }
```

Reply:
> Don't we have to transform the following one?
> 
> ```js
>  &--error {
>     color: $critical;
>   }
> ```

The missing ones are not used and that is why you got rid of them, no?

Reply:
yep exactly. no one's using them - time to get rid 🧹